best-practices/tagging/index.html

```diff
@@ -303,15 +333,14 @@ <h4>Headings</h4>
 				</aside>
 				<aside class="example" title="Multiline heading">
 					<p>Note that while <code>&lt;br/></code> can be used to force multiple lines, doing so will hardcode those breaks. It is better to identify the heading with either a tag or class and use CSS to center the braille &lpar;<code>text-wrap: balance</code>&rpar;. See the <a href=https://daisy.github.io/ebraille/best-practices/styling/#line-breaking>Styling Best Practices section on line breaks</a> for more information.</p>
```

Comment:
> Note that while `<br/>`can be ...

Now that the example does not contain `<br/>` anymore, people will wonder what this paragraph means.

Reply:
I tried to address this in https://github.com/daisy/ebraille/pull/288/commits/1ec23833f0d6e413bab8ba890cfb34a413fbf591.